ui/src/data/source.tsx

```diff
@@ -349,171 +340,121 @@ export class BackendUnderlaySource implements UnderlaySource {
   constructor(
     private underlaysApi: tanagra.UnderlaysApi,
     private exportApi: tanagra.ExportApi,
-    public underlay: Underlay,
-    public config: Configuration
+    public underlay: Underlay
   ) {}
 
-  lookupOccurrence(occurrenceID: string): Occurrence {
-    if (!occurrenceID) {
-      return { ...this.config.primaryEntity, id: "" };
-    }
-
-    return findByID(occurrenceID, this.config.occurrences);
-  }
-
-  lookupClassification(
-    occurrenceID: string,
-    classificationID: string
-  ): Classification {
-    return findByID(
-      classificationID,
-      this.lookupOccurrence(occurrenceID).classifications
+  lookupEntityGroup(entityGroupId: string): EntityGroupData {
```

Comment:
Would it be useful to add an endpoint for getting an entity group, similar to the existing `getEntity`? I think there's similar logic on the backend for lookup up a group from an entity(s).